docs/dependency-management.md

```diff
@@ -0,0 +1,105 @@
+# Dependency management
+
+-   Prefer `require` to `before`.
+-   Do not use `->` and `~>`, use `require`, `notify` or `subscribe`
+    instead.
+-   *Never* use `<-` or `<~`. (Lint will stop you, anyway.)
+-   Don't break encapsulation. In particular, a resource within one module
+    not create a dependency (require or notify) to a resource deep within
```

Comment:
should there be a 'should' in here or 'must' maybe?

Reply:
Yup, fixed